modules/getting-started/pages/docker.adoc

```diff
@@ -134,6 +134,19 @@ $ docker exec -it $(docker ps | grep tigergraph | awk '{print $1}') /bin/sh
 $ gadmin start all
 ----
 
+. Attend to security.
++
+WARNING: Your TigerGraph image is preconfigured with a superuser having the default username and password `tigergraph`. If you do not change this, anyone with access to your database will be able to read and modify it.
++
+To xref:tigergraph-server:user-access:user-management.adoc#_change_a_users_password[change the password] of the user called `tigergraph`:
```

Comment:
There are two passwords to change,
Linux user "tigergraph" with "tigergraph" password.  We can point them to the linux utility 'passwd' or just mention to use the Operating System tools to change the default password.
TigerGraph Application user "tigergraph" with "tigergraph" password.   Pointing them to "TigerGraph database user management" link is nice.

Reply:
Ok. The need to change both users (Linux and TigerGraph DB) was not clear to me, so I didn't write that.